build/Build.Steps.Linux.cs

```diff
@@ -22,7 +20,7 @@ partial class Build
                 arguments: $"../ -DCMAKE_BUILD_TYPE=Release -DOTEL_AUTO_VERSION={VersionHelper.GetVersionWithoutSuffixes()} -DOTEL_AUTO_VERSION_MAJOR={major} -DOTEL_AUTO_VERSION_MINOR={minor} -DOTEL_AUTO_VERSION_PATCH={patch}",
                 workingDirectory: buildDirectory);
             Make.Value(
-                arguments: $"",
+                arguments: $" ",
```

Comment:
https://github.com/nuke-build/nuke/issues/1417

Reply:
Can you add TODO comments and a tracking issue to revert it once the bug is fixed?

Reply:
a1e1bcb8a3761ca43f1767506a8c66c4ac81d36b